### DISCUSSION TITLE
Portable version?

### DISCUSSION THREAD

**Discussion by Ygarr** [Floorp-Projects/Floorp#1156] [Feature Request]
Suggest also release portable version to just unzip or un7z and use? Or officially delegate the function to someone

**Comment by creeper-0910** [comment]
https://github.com/Floorp-Projects/Floorp-Portable

**Reply by creeper-0910** [reply]
We recommend that you visit the official website (floorp.app) before posting a message in the discussion.
Even the smallest thing can save the developer's time, so please help us.
